test/unit/utils-test.js

```diff
@@ -145,14 +147,94 @@ suite('Utils', () => {
       vendorName: 'LIFX',
       productName: 'White 800 (Low Voltage)',
       productFeatures: {
+        chain: false,
         color: false,
         infrared: false,
-        multizone: false
+        multizone: false,
+        ['temperature_range']: [2700, 6500]
```

Comment:
We should probably also add a test for the Day & Night bulbs from 1500 to 4000K.

Reply:
Agreed, in the next pr which cleans the kelvin matter.